tests/functional/docs/test_glacier.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from tests.functional.docs import BaseDocsFunctionalTest
+
+class TestGlacierDocs(BaseDocsFunctionalTest):
+    def test_account_id(self):
+        self.check_autopopulated_param_doc(
+            'glacier', 'abort_multipart_upload', 'accountId')
+
+    def test_checksum(self):
+        self.check_autopopulated_param_doc(
```

Comment:
This is a vague method name that makes it hard to understand what's being tested here.

Is this asserting anything? (maybe it should start with assert_)?  What exactly about the autopopulated param doc are you checking.

Perhaps some combination of renaming the method and kwarg usage can make this more clear what's going on.